react-native/use_http_links_in_react_native.mo.md

```diff
@@ -0,0 +1,76 @@
+# [MO] Use HTTP links with RN Components/API Calls on iOS
+## Owner: [Julien Nassar](https://github.com/juliennassar)
+## Sponsor: [Alexandre Moureaux](https://github.com/almouro)
+
+## Prerequisites
+- [ ] Have a React Native application
```

Comment:
Have a React Native application **(using + link to CRNA or other dev standard)**